README.md

```diff
@@ -77,7 +81,7 @@ All the available Tomcat images are built on top of the `docker.io/xenit/openjdk
 
 ### How to build
 
-Release builds are produced by [Travis](https://travis-ci.org/xenit-eu/) driving Gradle from a `.travis.yml` file.
+Builds are produced by [GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions) driving Gradle from a GitHub workflow file.
```

Comment:
Can we link to the Actions tab of this repository instead of linking to github docs?